misc/pause-container/Makefile

```diff
@@ -20,7 +20,7 @@ TAG = 0.1.0
 all: container
 
 container:
-	docker build . -t "$(IMAGE):$(TAG)"
```

Comment:
The build should succeed even without this patch. 
But you can change it if necessary.